.github/workflows/build.yml

```diff
@@ -167,14 +184,24 @@ jobs:
           # Password for decrypting `mac_certs`
           mac_certs_password: ${{ secrets.mac_certs_password }}
 
-  finalize:
-    needs: [upload_artifacts]
-    if: always()
-    # && ${{ needs.upload_artifacts.result == 'skipped' || needs.upload_artifacts.result == 'success' }}
+      - name: upload artifacts to github actions
+        uses: actions/upload-artifact@v3
+        with:
+          name: build-${{matrix.os}}
+          path: |
+            app/dist/*.deb
+            app/dist/*.pacman
+            app/dist/*.AppImage
+            app/dist/*.rpm
+            app/dist/*.dmg
+            app/dist/*.zip
+            app/dist/*.exe
```

Comment:
Do we need this? Electron builder should upload the build results to a draft pre-release automatically

Reply:
This is helpful for getting the artifacts of prs